### ISSUE TITLE
Database manager incorrectly registers the hodoscopeConverter.

### ISSUE THREAD

**Issue by mholtrop** [JeffersonLab/hps-java#508]
Incorrect logic to only instantiate depending on run number, but then try to register all drivers. This results in registering a null pointer for low run numbers, causing a crash.

**Comment by JeremyMcCormick** [comment]
@mholtrop I thought you fixed this with a prior PR.  Did it get reverted?

**Comment by JeremyMcCormick** [comment]
The fix is simple I think.  

In DatabaseConditionsManager you just need to change line 761:

```
this.registerConditionsConverter(this.hodoscopeConverter);
```

to

```
if (this.hodoscopeConverter != null) {
    this.registerConditionsConverter(this.hodoscopeConverter);
}
```

**Comment by JeremyMcCormick** [comment]
I thought we fixed this already but it is now resolved (again?) by PR #510.
